src/main/resources/com/google/api/codegen/php/standalone_sample.snip

```diff
@@ -0,0 +1,65 @@
+@extends "php/common.snip"
+@extends "php/method_sample.snip"
+
```

Comment:
nit: You have some trailing whitespace in several lines in this file. Please remove it if you can.

Reply:
Ack